app/main.py

```diff
@@ -1 +1,33 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(
+            self,
+            name: str,
+            health: int = 100,
+            hidden: bool = False
+    ) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = hidden
+        Animal.alive.append(self)
+
+    def alive_animal(self, damage: int) -> None:
```

Comment:
In case you would like to create a method, it should represent action. Rename `alive_animal` to `decrease_health` or use another name.